python/ctsm/subset_data.py

```diff
@@ -571,6 +571,10 @@ def setup_files(args, defaults, cesmroot):
         abort("inputdata directory does not exist")
 
     # DATM data
+    # TODO issue #1895: allow datm_crujra, which also affects
+    #      tools/site_and_regional/default_data_1850.cfg
+    #      tools/site_and_regional/default_data_2000.cfg
+    #      python/ctsm/test/testinputs/default_data.cfg
```

Comment:
Check whether subset_data expects datm monthly files. CRUJRA2024 is in annual files.